android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/TaskExtension.kt

```diff
@@ -29,4 +29,17 @@ fun Task.hasStarted() =
     this.executionPeriod.hasStart() &&
     with(this.executionPeriod.start) { this.before(today()) || this.isToday() }
 
+fun Task.isReady() =
+  this.hasExecutionPeriod() &&
```

Comment:
wouldn't you want the end to be in future? this reads like it's checking that the end is in the past (or is today)

Reply:
Yeah, this makes sense. It wasn't super clear initially from the scoping description in the associated issue.

Reply:
Yea true good point, I think the issue spec is wrong, the ways tasks will be written it'll be valid between some start and end time (I think? @AnnieMungai @dubdabasoduba you can comment to verify), in the abstract that at least makes sense, I'll correct #1701 too